source/linux/Foundational_Components/Kernel/Kernel_Drivers/Display/DSS7.rst

```diff
@@ -663,6 +665,58 @@ The following example was run on DSS7-UL and hence the pipe with scaling capabil
 
 Note that the ``*2`` at the end of ``-P 41@38:400x400*2`` is the scaling factor.
 
+- **Cropping**
+
+``kmstest`` utility can be used to demonstrate cropping in a video frame using a test pattern.
+User can specify the video frame size ``w x h`` with an input color format ``cf`` using option ``-f``, then to specify the source rectangle i.e. to create cropped portion of video frame, it can be done using ``-v`` argument which specify ``w1 x h1`` source rectangle starting at coordinates ``x1,y1`` w.r.t the video frame. The destination rectangle i.e. on the screen where this needs to be displayed can be specified using ``-p`` option with the desired dimensions i.e. ``w2 x h2`` to be displayed starting at coordinates ``x2,y2``.
+
+.. code-block:: console
+
+   $ kmstest -p <plane_number>:<x-coordinate>,<ycoordinate>-<plane_width>x<plane_height> -f <frame_width>x<frame_height>-<color-format> -v <view_number>:<x-coordinate>,<ycoordinate>-<view_width>x<view_height>
+
+   $ kmstest -p 0:x2,y2-w2xh2 -f wxh-cf -v x1,y1-w1xh1
```

Comment:
This is not a valid command and is just more confusing than the immediate above line where all parameters are explained.

Reply:
Should I remove these commands??
Examples are good to explain I think.